DataHub/UserInterface/03_ETLExtensions.md

```diff
@@ -0,0 +1,1764 @@
+| Integer                          | Base price                     |
+| Float                            | Base price                     |
+
+### Configuration
+
+This ETL extension has no further configuration settings.
+
+
+
+## PIM Price to Simple Price
+
+![PIM price to simple price](../../Assets/Screenshots/DataHub/Settings/ETL/Extensions/PIMPriceToSimplePrice.png "[PIM price to simple price]")
+
+This extension is used to map a PIM price field attribute to a simple price field attribute. The PIM price field is Actindo specific and contains not only the simple price, but also the base price, scale prices, promotion prices, the tax class including the tax rate and supported currencies. In this case, all values that are also included to the simple price field attribute are mapped. Further settings are defined in the configuration.
+
+ [comment]: <> (Is that correct? Which values? Base price, scale prices and indication about net or gross price?)
+
+### Possible data type mappings
+
+| Destination attribute data type  | Source attribute data type     |
+|----------------------------------|--------------------------------|
+| Simple Pricing                   | Price field                    |
+
+### Configuration
+
+- *Country*     
+  Select the appropriate country in the drop-down list to apply the correct tax rate to the price. All available countries are displayed in the list.
+
+- *Mapping mode*   
+  Select the appropriate mapping mode for the PIM price field attributes. The following modes are available:   
+    - **Consider all prices**: All prices in the PIM price field attribute are mapped to the destination attribute.
+    - **Consider only promotions**: Only the promotion prices in the PIM price field attribute are mapped to the destination attribute.
+    - **Ignore promotions**: All prices beside the promotion prices in the PIM price field attribute are mapped to the destination attribute.
+
+[comment]: <> (Is that correct? Why country? Is a tax class or rate mapped? Where are the promotions prices mapped to: scale prices???)
+
+
+
+## Preg Replace Extension
+
+![Preg replace extension](../../Assets/Screenshots/DataHub/Settings/ETL/Extensions/PregReplaceExtension.png "[Preg replace extension]")
+
+This extension is used to replace the value of a source attribute by a certain value and map it to a destination attribute. The replacement settings are defined in the configuration.
+
+### Possible data type mappings
+
+| Destination attribute data type  | Source attribute data type         |
+|----------------------------------|------------------------------------|
+| String                           | String, Text field, Integer, Float |
+| Text field                       | String, Text field, Integer, Float |
+
+### Configuration
+
+- *Pattern*   
+  Enter a regular expression for the source attribute value that should be replaced. For detailed information about regular expressions, see https://regex101.com/.
```

Comment:
In Taxes haben wir diesen Satz verwendet:  The site https://regex101.com/ can be useful to test regular expressions.